### PR TITLE
Aggregate more columns in more ways from the historical transactions.

### PR DESCRIPTION
**Pull request by erwald**
Later, I think we can do exactly the same operations for the
new_merchant_transactions table, as well.

I also noticed that a weight decay of 0.5 was better for avoiding
overfitting. In general, I believe one should use a higher weight
decay the more features one has.

### CODE REVIEW

File: elo_loyalty_prediction.ipynb [hunk 1]
```diff
@@ -741,7 +819,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.2"
+   "version": "3.7.1"
```
Comment (erwald) [inline elo_loyalty_prediction.ipynb]: I'll make sure to remember to upgrade Python on my work laptop tomorrow. 😅
